app/models/user_preference.rb

```diff
@@ -3,8 +3,12 @@ class UserPreference
 
   def initialize
     array = []
-    Metacode.all.find_each do |m|
-      array.push(m.id.to_s)
+    begin
+      Metacode.all.find_each do |m|
+        array.push(m.id.to_s)
+      end
+    rescue ActiveRecord::StatementInvalid
+      Rails.logger.warn("TODO: remove this travis workaround in user_preference.rb")
```

Comment:
I think if you do

bundle exec rake db:setup

Instead of

bundle exec rake db:schema:load

In Travis, it will load the db/seeds and fix the issue.